modules/apps/map/map-openstreetmap/package.json

```diff
@@ -1,5 +1,6 @@
 {
 	"dependencies": {
+		"@liferay/map-common": "*",
 		"metal-state": "2.16.8"
 	},
 	"name": "map-openstreetmap",
```

Comment:
Maybe namespace this too like a good boyscout?

![boyscout](https://user-images.githubusercontent.com/905006/135218181-22a79def-4184-47ae-89b7-3a7a5feae63c.gif)